kol/onshape/schema/part.py

```diff
@@ -17,7 +17,6 @@ class PartMetadata(BaseModel):
     jsonType: str
     isMesh: bool
     partId: str
-    partIdentity: str
```

Comment:
Not present in all parts raises error with pydantic and not used anywhere.

Reply:
can use optional pydantic fields